bootstrap.sh

```diff
@@ -27,6 +27,9 @@ pip install ansible
 ## Install Boto3
 pip install boto3
 
+## Install Ruby SDK for AWS Directory Data API Services
+gem install aws-sdk-directoryservicedata
```

Comment:
I don't really understand how this is going to be used, but would it make sense to use [bundler inline](https://bundler.io/guides/bundler_in_a_single_file_ruby_script.html) in whatever script you run to make the changes? That would "lazily" install the gem so that we don't have it on every Buildkite instance, just the ones running that script, which seems nice to me.